src/ProxyQuery/BaseProxyQuery.php

```diff
@@ -62,37 +62,25 @@ public function __call(string $name, array $args)
         return \call_user_func_array([$this->queryBuilder, $name], $args);
     }
 
-    /**
-     * @param mixed $sortBy
```

Comment:
can you replace for the correct param? In all cases (string|null)

Reply:
This was moved to the interface